Makefile

```diff
@@ -284,7 +284,7 @@ push-image: build-image
 
 .PHONY: local
 ## Local: Run operator locally
-local: deploy-clean deploy-rbac deploy-crds deploy-cr
+local: deploy-clean deploy-rbac deploy-crds #deploy-cr
```

Comment:
I think you can really remove this target, instead of just commenting it out.

Reply:
True 🤖

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/123/commits/a550fa1817e6f8d293f2354344619909fac1e445